tests/vmi_kernel_boot_test.go

```diff
@@ -71,8 +66,7 @@ var _ = Describe("[sig-compute]VMI with external kernel boot", decorators.SigCom
 
 		It("ensure successful boot and deletion when VMI has a disk defined", func() {
 			By("Creating VMI with disk and kernel boot")
-			vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
-			vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1Gi")
+			vmi := libvmifact.NewAlpine(libvmi.WithResourceMemory("1Gi"))
```

Comment:
It is not a big problem and maybe you will not want to handle it here, but it will be nice to investigate why a 1G memory was set (instead of the default 256/512M).

Reply:
ok. trying